addons/website/views/snippets/snippets.xml

```diff
@@ -44,7 +44,7 @@
                     <t t-snippet="website.s_numbers" string="Numbers" t-thumbnail="/website/static/src/img/snippets_thumbs/s_numbers.svg">
                         <keywords>statistics, stats, KPI</keywords>
                     </t>
-                    <t t-snippet="website.s_picture" string="Picture" t-thumbnail="/website/static/src/img/snippets_thumbs/s_picture.svg">
+                    <t t-snippet="website.s_picture" string="Title - Image" t-thumbnail="/website/static/src/img/snippets_thumbs/s_picture.svg">
                         <keywords>image, media, illustration</keywords>
```

Comment:
A cool bonus would be to add the "old" name in the keywords so that our historical users can still type "Picture" in the search bar.